packages/framework/tests/Unit/Component/Translation/ConstraintMessagePropertyExtractorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrameworkBundle\Unit\Component\Translation;
+
+use JMS\TranslationBundle\Model\FileSource;
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Model\MessageCatalogue;
+use PhpParser\ParserFactory;
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Translation\ConstraintMessagePropertyExtractor;
+use SplFileInfo;
+
+class ConstraintMessagePropertyExtractorTest extends TestCase
+{
+    public function testExtractFromConstraint(): void
+    {
+        $file = new SplFileInfo(__DIR__ . '/MyConstraintClass.php');
```

Comment:
I didn't get meaning of prefix `My`. I suggest to rename these classes to `ConstraintClass` and `NonConstraintClass`